spec/metadata-1.1.html

```diff
@@ -93,26 +93,34 @@ <h2>FPCore metadata properties</h2>
     synonymous with such a list.
 
     <dl class="code-terms">
-      <dt>(float <var>e</var> <var>s</var>)</dt>
+      <dt>(float <var>e</var> <var>nbits</var>)</dt>
       <dd>An IEEE-754 binary floating-point value with <var>e</var>
-      exponent bits and <var>s</var> significant bits (including the
-      implicit bits). <code>binary64</code> and <code>binary32</code>
-      are common shorthands.</dd>
+      exponent bits and <var>nbits</var> total bits.
+      There will be <var>nbits</var> - <var>e</var> significand bits
+      (including the implicit bit).
+      <code>binary64</code> and <code>binary32</code> are common shorthands.</dd>
 
       <dt>(posit <var>es</var> <var>nbits</var>)</dt>
       <dd>Posits with <var>es</var> exponent bits and <var>nbits</var>
       total bits. <code>posit8</code>, <code>posit16</code>,
       <code>posit32</code>, and <code>posit64</code> are common shorthands.</dd>
 
-      <dt>(fixed <var>i</var> <var>f</var>)</dt>
-      <dd>Fixed point, with <var>i</var> integer bits and <var>f</var>
-      fractional bits.</dd>
+      <dt>(fixed <var>scale</var> <var>nbits</var>)</dt>
+      <dd>Fixed point, with <var>nbits</var> total bits
+      and a scale factor of 2 ^ <var>scale</var>.
+      One bit will be used to store the sign.
+      For example, (fixed 0 8) can represent the 8-bit integers from -127 to 127,
+      while (fixed -8 8) can represent magnitudes from -127/128 to 127/128 in steps of 1/128.
+      Overflow behavior is determined by the
+      <a href="metadata-1.1.html#p:fixed-overflow">fixed-overflow</a> property.</dd>
 
       <dt>real</dt>
       <dd>A special precision for arbitrary real numbers, where no rounding takes place.</dd>
 
       <dt>integer</dt>
-      <dd>A special precision for mathematical integers, where no rounding takes place.</dd>
+      <dd>A special precision for mathematical integers, where no rounding takes place.
+      Implementations can assume that all values with this precision are indeed integers:
+      if a non-integer value is computed, implementations should signal an error.</dd>
```

Comment:
Should or may signal an error? Not that it matters much. I'd rather write an ambiguous `can` since we don't have any implementations of this yet.